Lib/test/test_imaplib.py

```diff
@@ -468,16 +468,6 @@ def test_logincapa(self):
             _server = self.imap_class(self.host, self.port)
             self.check_logincapa(_server)
 
-    def test_logincapa_with_client_certfile(self):
```

Comment:
> @larryhastings: Please see my summary of this PR, #2475 (comment)
> 
> Technically, I don't think that you can merge a PR if the CI fails. So it's not posisble to add the CI config in one PR and then fix failing tests. I could do the opposite, but it's not possible to make a PR depends on another one. Moreover, I used this issue to see which tests on the CI and test my fixes.

I think this reply below was actually meant to be in-line here.  I am replying to it here.

2bef165 says "Remove the two tests which are already skipped."  If they're skipped, then they're not failing, and they don't need to be removed.

Are they actually failing?